contracts/claims/data/ethlive/data/airdrop.txt

```diff
@@ -0,0 +1,137 @@
+0xf68a2fb3fa4b90fba7c7700ed8c676cec94f3573 [49999999999, 0]
+0xd2c7008400f54aa70af01cf8c747a4473246593e [54000000000, 1000000]
+0x0079bf6868cb1b7cf4a451311fd70c348f3ed38b [1000000000000, 0]
+0x2fdeec247037a343b26f5daa0d3af15fe23a28c5 [328222496065, 0]
+0xb8a6b2a4c492eb20686ac1706bc5a2548a9e4000 [34220000000000, 0]
+0x7ef3d3d72c3cdd897c6560677cc91f4237ae7443 [50000000000, 0]
+0xd11daeda7061bef1955129b76a1dd2162da43c51 [0, 1000000000]
+0x2dcd241135d2d5b2e9ba2926e436c054fdf8bf0b [50000000000, 0]
+0xb196740f7742480ccb639462e35aabb1944bc02a [1000000000000, 0]
+0xe10208aac0f0d1b0ba62a1e65ce6728b0349370c [200000000000, 0]
+0x4d5bdd096071efee163f8b2eae7a21c9686d3ff3 [55000000000, 0]
+0x50f80208161e10d228711c8fe4c12c56ca33c6a9 [2400000000000, 0]
+0x004ef7649d6589caab9edbb4994e6f0b785f18ac [15000000000000, 0]
+0x245b4775082c144c22a4874b0fba8c70c510c5ae [500000000000, 0]
+0xd90723ccf710e5248108dfc00c2b6cfd06ae0678 [1300983450000, 0]
+0x5451d4a6e38f2710e18e0b4f339c678cad6f4784 [40983450000, 0]
+0x30d0424b0ef581aa8e87e5865679559673b33ee6 [98949983400000, 2598224506]
+0x42d9384400a50755f049817b88541bb0ec5c3aab [19983500000, 0]
+0x67cd3abe59504c7886c946941b3218c995779c8e [49983500000, 0]
+0xb072244f837da64271f9c59f713390471633970c [16529983400000, 36400037528]
+0x830b5d9f275407863a0d7875155e764cc0e56463 [49983500000, 0]
+0x056815e3c8538cd34be4c06dac9ae300db043b53 [200024983400000, 0]
+0x770b38de1dc5b8f3432e33e2df85b70b0d3f14b6 [2800000000000, 0]
+0x0aed302952fbdb02681e62d7d0b6046bc1c409ca [149568783000000, 0]
```

Comment:
149568783000000 is the amount of ERC20 RING? what is decimals?

Reply:
this decimals is 9